frontend/src/pages/projects/screens/detail/connections/ConnectionsList.tsx

```diff
@@ -67,17 +69,30 @@ const ConnectionsList: React.FC = () => {
           iconImage={typedEmptyImage(ProjectObjectType.connections)}
           imageAlt="create a connection"
           createButton={
-            <Button
-              key={`action-${ProjectSectionID.CONNECTIONS}`}
-              variant="primary"
-              data-testid="create-connection-button"
-              onClick={() => {
-                setManageConnectionModal({});
-              }}
-              isDisabled={enabledConnectionTypes.length === 0}
-            >
-              Create connection
-            </Button>
+            <>
+              <Button
+                key={`action-${ProjectSectionID.CONNECTIONS}`}
+                variant="primary"
+                data-testid="create-connection-button"
+                aria-describedby={
+                  enabledConnectionTypes.length === 0 ? 'no-connection-types-tooltip' : undefined
+                }
+                isAriaDisabled={enabledConnectionTypes.length === 0}
+                onClick={() => {
+                  setManageConnectionModal({});
+                }}
+                ref={tooltipRef}
+              >
+                Create connection
+              </Button>
+              {enabledConnectionTypes.length === 0 && (
+                <Tooltip
+                  id="no-connection-types-tooltip"
+                  content="No connection types available"
+                  triggerRef={tooltipRef}
+                />
+              )}
```

Comment:
We also have an `Add connection` that gets disabled just the same if there's no connection types. But right now there's no tooltip.
@simrandhaliw i think this button should also be named `Add connection`, no? Other pages don't mix `create` and `add` actions labels.

Reply:
All instances of connection creation should use "Create" instead of "Add," since the opposite action is "Delete". If the opposite action were "Remove," then "Add" would be appropriate.

Also, we should update the preview drawer's modal heading from "Add connection" to "Create connection," as this is consistent with the terminology used for connection creation modals.

Reply:
> We also have an Add connection that gets disabled just the same if there's no connection types. But right now there's no tooltip.

@christianvogt  Do you mean the "Create connection" action in the Connections tab of the Project details view? Yes, that is another instance where the same tooltip would be displayed for the disabled button.

Reply:
There's several inconsistencies with other tabs. Perhaps it's best to create an issue to align them all instead of part of this PR.